src/domain/individualApplication/queries.ts

```diff
@@ -1,11 +1,18 @@
 import { gql } from 'apollo-boost';
 
 export const FILTERED_CUSTOMERS_QUERY = gql`
-  query FILTERED_CUSTOMERS($firstName: String, $lastName: String) {
+  query FILTERED_CUSTOMERS(
+    $firstName: String
+    $lastName: String
+    $emails_Email: String
```

Comment:
>$emails_Email
>$addresses_Address

Would it make sense to change these variables to simply `email` and `address`? Wouldn't need the switch in `IndividualApplicationPageContainer`.

Reply:
Good point 👍 
Fixed!